src/main/scala/org/broadinstitute/dsde/firecloud/EntityClient.scala

```diff
@@ -340,4 +349,43 @@ class EntityClient (requestContext: RequestContext, modelSchema: ModelSchema)(im
       }
     }
   }
+
+  def importPFB(pipeline: WithTransformerConcatenation[HttpRequest, Future[HttpResponse]],
+                workspaceNamespace: String, workspaceName: String, pfbRequest: PfbImportRequest): Future[PerRequestMessage] = {
+
+    def callArrow(url: String) = {
+      pipeline { Post(avroToRawlsURL, PfbImportRequest(url)) }
+    }
```

Comment:
consider having  `callArrow` accept a `PfbImportRequest`, a `URL`, or a `URI` instead of a `String` as its argument

Reply:
Now that I'm leaving it to Arrow to sanitize the URL rather than doing that here, I can remove the argument to `callArrow` and let it use the `pfbRequest` already in scope.